src/main/java/org/terasology/smithing/event/ProduceCharcoalRequest.java

```diff
@@ -18,6 +18,9 @@
 import org.terasology.entitySystem.event.Event;
 import org.terasology.network.ServerEvent;
 
+/*
+* Sends a request to begin producing charcoal
```

Comment:
Somewhat like the prior note: It is more like this is _being_ sent or being used as part of the process, the event itself isn't sending anything, it is being sent :-)